tests/test_app.py

```diff
@@ -32,6 +32,10 @@ def test_post_with_token(client):
     assert 'csrf_token The CSRF token is missing' not in str(response.data)
     # token is still invalid :(
 
+def test_no_message_customization(client):
+    response = client.get('/secret', follow_redirects = True)
```

Comment:
Minor detatil on code style:

```suggestion
    response = client.get('/secret', follow_redirects=True)
```